### ISSUE TITLE
Retrieve raw message data from Apache Pulsar

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#21]
## User story
1. As a user
2. I want to read all messages regarding a specific/all topic(s)
3. So that can get an overview about the topics raw data.

## Acceptance criteria
* [x] User can select a topic
* [x] Program prints a reasonable number of raw messages (including metadata if available)
* [x] If no topic is selected, a reasonable number of messages from all topics are displayed

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
